umi_gripper/umi_gripper.xml

```diff
@@ -0,0 +1,141 @@
+<mujoco model="umi_model">
+
+  <compiler angle="radian" meshdir="assets/" texturedir="assets/" />
+
+  <!-- Simulation Options -->
+  <option integrator="implicitfast" impratio="1" cone="elliptic" noslip_iterations="2">
+    <flag multiccd="enable" />
+  </option>
+
+  <!-- Assets -->
+  <asset>
+    <mesh name="base_link" file="base_link.STL" />
+    <mesh name="finger_holder_left" file="finger_holder_left.STL" />
+    <mesh name="finger_left" file="finger_left.STL" />
+    <mesh name="finger_holder_right" file="finger_holder_right.STL" />
+    <mesh name="finger_right" file="finger_right.STL" />
+    <mesh name="gopro" file="gopro.STL" />
+    <material name="mirror" rgba="0 0 0 1" texrepeat="5 5" reflectance="1.0" />
+    <texture name="right_aruco_sticker" type="2d" file="right_aruco_sticker.png" />
+    <texture name="left_aruco_sticker" type="2d" file="left_aruco_sticker.png" />
+    <material name="Aruco_Right_Base_Sticker" texture="right_aruco_sticker" />
+    <material name="Aruco_Left_Base_Sticker" texture="left_aruco_sticker" />
+  </asset>
+
+  <!-- Default settings -->
+  <default>
+    <default class="umi_model">
+      <!-- Joint Defaults -->
+      <joint frictionloss="0.1" armature="0.1" />
+
+      <!-- Gripper Joints -->
+      <default class="gripper_slide">
+        <joint type="slide" damping="0.1" range="-1.2 1.2" />
+        <position kp="100" dampratio="1" ctrlrange="-1.2 1.2" />
+      </default>
+      <default class="gripper_hinge">
+        <joint type="hinge" damping="10.1" range="-31.415 31.415" />
+        <position kp="10" dampratio="0.1" ctrlrange="-3.1415 3.1415" />
+      </default>
+
+      <!-- Finger Joints -->
+      <default class="finger_slide">
+        <joint type="slide" stiffness="100" damping="10.1" range="0 0.05" />
+      </default>
+
+      <!-- Actuator Defaults -->
+      <position kp="1000" />
+      <default class="finger_actuator">
+        <position ctrlrange="0 0.05" />
+      </default>
+      <default class="gripper_actuator">
+        <position kp="100" dampratio="1" ctrlrange="-1.2 1.2" />
+      </default>
+      <default class="gripper_rot_actuator">
+        <position kp="10" dampratio="0.1" ctrlrange="-3.1415 3.1415" />
+      </default>
+
+      <!-- Geom Defaults -->
+      <default class="marker">
+        <geom type="box" size="0.009 0.009 0.00001" euler="0 0 0" rgba="1 1 1 1" contype="0" conaffinity="0" mass="0.0" />
+      </default>
+      <default class="finger_holder_geom">
+        <geom type="mesh" contype="0" conaffinity="0" group="1" density="0" rgba="1 1 1 1" />
+      </default>
+
+    </default>
+  </default>
+
+  <worldbody>
+
+    <body pos='0 0 0' euler="-1.571 0 0">
```

Comment:
* Remove pos='0 0 0' (it's the default)
* Change euler to quat for consistency if possible.